filebeat/docs/filebeat-options.asciidoc

```diff
@@ -511,6 +512,25 @@ The `enabled` option can be used with each prospector to define if a prospector
 
 When used with `type: udp`, specifies the maximum size of the message received over UDP. The default is 10240.
 
+[float]
+[[line-delimiter]]
+==== `line_delimiter`
+
+When used with `type: tcp`, specifies the the characters used to split the incoming events. The default is '\n'.
+
+[float]
+[[max-read-buffer]]
+==== `max_read_buffer`
```

Comment:
We have `max_message_size` in udp. We should probably use the same config name. Also we can change the udp one.

Reply:
I will rename `max_read_buffer` to `max_message_size`, after looking back at the implementation they are a bit different but the underlying concept is the same, _making sure we don't OOM on incoming messages_